plugins/rbac/src/components/CreateRole/CreateRolePage.tsx

```diff
@@ -15,7 +39,25 @@ export const CreateRolePage = () => {
       <Page themeId="tool">
         <Header title="Create role" />
         <Content>
-          <CreateRoleForm />
+          <RoleForm
+            initialValues={initialValues}
+            titles={{
+              formTitle: 'Create Role',
+              nameAndDescriptionTitle: 'Enter name and description of role ',
+              usersAndGroupsTitle: 'Add users and groups',
+              permissionPoliciesTitle: '',
+            }}
+            membersData={{
+              members: Array.isArray(members)
```

Comment:
So when `members` isn't an array, what can it be?  A `Response` object that might contain an error from the API?

Reply:
yes a Response object, and the object will have the status code and the status message